### PR TITLE
Add podspec & improve docs

### PR DESCRIPTION
**Pull request by radex**

### CODE REVIEW

**Comment by radex** [comment]
‼️ DO NOT MERGE - dependent branch ‼️

File: docs-master/Installation.md [hunk 1]
```diff
@@ -45,43 +45,45 @@ npm install @nozbe/with-observables
...
-    If you don't want to use `react-native link`, you can link the library manually:
+3. **Link WatermelonDB's native library with the Xcode project -- manually**:
 
     1. Open your project in Xcode, right click on **Libraries** in the Project Navigator on the left and click **Add Files to "Your Project Name"**. Look under `node_modules/@nozbe/watermelondb/native/ios` and select `WatermelonDB.xcodeproj`
     2. Go to Project settings (top item in the Project navigator on the left), select your app name under **Targets** → **Build Phases** → **Link Binary With Libraries**, and add `libWatermelonDB.a`
 
     For more information about linking libraries manually, [see React Native documentation](https://facebook.github.io/react-native/docs/linking-libraries-ios).
 
-    **Using CocoaPods**
+4. **Link WatermelonDB's native library with the Xcode project -- using CocoaPods**:
```
Comment (michalpopek) [inline docs-master/Installation.md:55]: For a few seconds I was convinced that I need to do both steps 3 and 4 here 😉 IMO the previous instructions were a bit more clear about this matter

Comment (radex) [reply docs-master/Installation.md:55]: thanks, I'll improve this

File: docs-master/Installation.md [hunk 2]
```diff
@@ -45,43 +45,45 @@ npm install @nozbe/with-observables
...
-    [Please contribute!](https://github.com/Nozbe/WatermelonDB/issues/279)
+        ```ruby
+        # Before:
+        pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
+        # Change to:
+        pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi', :modular_headers => true
+        ```
 
-Note that Xcode 9.4 and a deployment target of at least iOS 9.0 is required (although Xcode 10 and iOS 11.0 are recommended).
+Note that Xcode 9.4 and a deployment target of at least iOS 9.0 is required (although Xcode 13.3+ and iOS 12.0+ are recommended).
```
Comment (djorkaeffalexandre) [inline docs-master/Installation.md:71]: I guess Xcode 13 was not launched yet hehehehe

Comment (djorkaeffalexandre) [reply docs-master/Installation.md:71]: Happy to made my first _(small)_ contribution :)
https://github.com/Nozbe/WatermelonDB/pull/748

**Comment by sidferreira** [comment]
@radex plans to deploy 0.19?

**Pull request merged**